refinery_core/src/runner.rs

```diff
@@ -230,7 +230,12 @@ impl Runner {
         }
     }
 
-    /// set the target version up to which refinery should migrate, Latest migrates to the latest version available
+    /// Get the known migrations.
+    pub fn get_migrations(self) -> Vec<Migration> {
```

Comment:
we don't want to consume the Runner, should be `&self`

Reply:
And I'm guessing


    pub fn get_migrations(&self) -> &Vec<Migration> {
        &self.migrations
    }

is better than

    pub fn get_migrations(&self) -> Vec<Migration> {
        self.migrations.clone()
    }

(new to Rust :))

Reply:
yeah, no need to clone, let the caller of the function decide if they want to call clone